imagemodal/public/view.js

```diff
@@ -45,7 +61,7 @@ function ImageModalView(runtime, element) {
         draggie = draggie || new Draggabilly(
             image[0],
             {
-                containment: true
+                containment: true,
```

Comment:
These trailing commas were actually added automatically;
if you invoke `eslint` with the `--fix` flag, it will fix anything it can,
like trailing commas, single vs double quotes, etc.!